internal/chain/util.go

```diff
@@ -21,3 +21,12 @@ func IsValidAddress(address string, checksummed bool) bool {
 	}
 	return !checksummed || common.HexToAddress(address).Hex() == address
 }
+
+func LSKToWei(amount int64) *big.Int {
+	lsk := new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
+	return new(big.Int).Mul(big.NewInt(amount), lsk)
+}
```

Comment:
Wei is not a LSK denomination. Should we update the name to something more appropriate?
Also, what does this method exactly do?

Reply:
Clarified offline.